### PR TITLE
revisi routing, tombol, sama masukin halaman ke bottom navigation, to…

### PR DESCRIPTION
**Pull request by Shinzi04**
Tambahin pages ke bottom navigation,
Revisi tombol back di myaccount,
ubah transfer history --> supaya sama navbarnya sama yang lain,
commnand beberapa hal yang gak guna (tolong di cek kalau misalnya bener2 tidak guna bisa dihapus juga)

### CODE REVIEW

File: lib/pages/myaccount/change_pin.dart [hunk 1]
```diff
@@ -28,11 +28,8 @@ class _ChangePinState extends State<ChangePin> {
           leading: BackButton(
```
Comment (felixfrdd) [inline lib/pages/myaccount/change_pin.dart:28]: [buildbackbutton](https://github.com/felixfrdd/abank_project/blob/8e0fd1856caaea47b76ba34ea496d8fe1f267210/lib/pages/transfer/new_account_page.dart#L22-L25)
colornya boleh transparent aja seperti di referensi link atas.

File: lib/pages/myaccount/set_a_limit.dart [hunk 2]
```diff
@@ -25,12 +25,7 @@ class SetALimit extends StatelessWidget {
           leading: BackButton(
```
Comment (felixfrdd) [inline lib/pages/myaccount/set_a_limit.dart:25]: [buildbackbutton](https://github.com/felixfrdd/abank_project/blob/8e0fd1856caaea47b76ba34ea496d8fe1f267210/lib/pages/transfer/new_account_page.dart#L22-L25)
colornya boleh transparent aja seperti di referensi link atas.

File: lib/pages/myaccount/view_cvv.dart [hunk 3]
```diff
@@ -18,12 +18,7 @@ class ViewCVV extends StatelessWidget {
           leading: BackButton(
```
Comment (felixfrdd) [inline lib/pages/myaccount/view_cvv.dart:18]: [buildbackbutton](https://github.com/felixfrdd/abank_project/blob/8e0fd1856caaea47b76ba34ea496d8fe1f267210/lib/pages/transfer/new_account_page.dart#L22-L25)
colornya boleh transparent aja seperti di referensi link atas.

**Review by felixfrdd — CHANGES_REQUESTED** [review]

File: lib/pages/myaccount/set_a_limit.dart (hunk 2 above)
Comment (Javier7898) [reply lib/pages/myaccount/set_a_limit.dart:25]: tttt

**Review by Javier7898 — APPROVED** [review]
ok 👍

**Review by IniGabriel — APPROVED** [review]
ok

**Review by felixfrdd — APPROVED** [review]

**Review by felixfrdd — APPROVED** [review]
later changes.

**Pull request merged**